tests/subsys/settings_commit_prio/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  settings.settings_commit_prio:
+    platform_allow: native_sim
+    tags: settings
```

Comment:
Can the `unit_testing` board not be used for this unit test?

Reply:
TBH the whole testing is a bit of a mystery to me. I run Twister locally to ensure I'm not pushing anything bad, but I am no Twister expert. How would this yaml file look like for the `unit_testing` board ? And what would be the local twister command line to verify?

Reply:
It will work pretty much the same. I don't know if there are any advantages of using the `unit_testing` board, but I usually use that for unit testing

Reply:
OK to leave it as is ?